Before: dd34a4f6ca3c2ff29021dcf53aeaafba13c7b3531a5b4002e734875ac6a45027
After: 9d6de05e909457c4714a14c22d8ae550c92ade09e7ceb69331556b282e636d87
Patch: 8b2b721d83fe7c443055aa4b00c262bf7a4ed8821f9fd66b5fdb29a1b38ba0e8
--- Gemfile	2012-07-22 20:44:49.468916014 +1000
+++ Gemfile	2012-07-22 21:27:47.320315586 +1000
@@ -1,5 +1,5 @@
 source :rubygems
 
-# Load Gemfile.corvid
-eval File.read(File.expand_path('../Gemfile.corvid',__FILE__))
+# Load Gemfile.raven
+eval File.read(File.expand_path('../Gemfile.raven',__FILE__))
 
--- Gemfile.corvid	2012-07-22 20:44:49.465582705 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,38 +0,0 @@
-# vim: ft=ruby et ts=2 sw=2 :
-header "# This file is generated by corvid update:deps"
-footer "# v\im: ft=ruby et ts=2 sw=2 :"
-
-if options[:use_corvid_gem]
-  gem 'corvid', ">= #{::Corvid::VERSION}"
-else
-  gem 'corvid', path: CORVID_ROOT
-end
-gem 'rake', group: :rake
-
-group :test do
-  gem 'simplecov'
-end if Dir.exists?('test')
-
-group :test_unit do
-  gem 'minitest'
-  gem 'guard-minitest', '>= 0.5.0'
-end if Dir.exists?('test/unit')
-
-group :test_spec do
-  gem 'rspec'
-  gem 'guard-rspec'
-end if Dir.exists?('test/spec')
-
-group :ci do
-  gem 'ci_reporter', require: false
-end
-
-group :doc do
-  gem 'yard'
-  gem 'rdiscount' # TODO aaaaaaaaaahhhhhhhhhh
-end
-
-group :debug do
-  gem 'debugger', platforms: :mri
-end
-
--- test/bootstrap/all.rb	2012-07-22 20:44:49.468916014 +1000
+++ test/bootstrap/all.rb	2012-07-22 21:27:47.320315586 +1000
@@ -1,4 +1,4 @@
 # encoding: utf-8
 APP_ROOT = File.expand_path('../../..',__FILE__)
 require 'rubygems'
-require 'corvid/test/bootstrap/all'
+require 'raven/test/bootstrap/all'
--- test/bootstrap/unit.rb	2012-07-22 20:44:49.468916014 +1000
+++ test/bootstrap/unit.rb	2012-07-22 21:27:47.320315586 +1000
@@ -1,4 +1,4 @@
 # encoding: utf-8
 $coverage_name = 'unit tests'
 require_relative 'all'
-require 'corvid/test/bootstrap/unit'
+require 'raven/test/bootstrap/unit'
